charts/scalardb-cluster/templates/scalardb-cluster/deployment.yaml

```diff
@@ -39,7 +42,11 @@ spec:
         - name: {{ .Chart.Name }}-node
           securityContext:
             {{- toYaml .Values.scalardbCluster.securityContext | nindent 12 }}
+          {{- if eq .Values.global.platform "azure" }}
+          image: "{{ .Values.global.azure.images.scalardbCluster.registry }}/{{ .Values.global.azure.images.scalardbCluster.image }}:{{ .Values.global.azure.images.scalardbCluster.tag }}"
+          {{- else }}
           image: "{{ .Values.scalardbCluster.image.repository }}:{{ .Values.scalardbCluster.image.tag | default .Chart.AppVersion }}"
+          {{- end }}
```

Comment:
We must pull container images from ACR for Azure Marketplace instead of our public container registry.
https://learn.microsoft.com/en-us/partner-center/marketplace-offers/azure-container-technical-assets-kubernetes#update-the-helm-chart